RHEL6_7/backup/UntrackedFiles/solution.txt

```diff
@@ -6,10 +6,10 @@ Some user data, such as user home directories and temporary files, are not track
 
 * The [link:./kickstart/untrackedexpected] file lists the regular files and symlinks created by runtime system operations (for example, handling runlevels, alternatives, and active SELinux modules). Most likely you do not need to care about them, the list is available just for the sake of completeness.
 
-* The [link:./kickstart/untrackedtemporary] file lists all temporary local files on the system that will not be migrated. This is essentially everything in the /cgroup/, /tmp/, and /var/ directories. Most likely you do not need to care about them, the list is available just for the sake of completeness.
+* The [link:./kickstart/untrackedtemporary] file lists all temporary local files on the system that will not be migrated. This is everything in the /cgroup/, /tmp/, and /var/ directories. Most likely you do not need to care about them, the list is available just for the sake of completeness.
 
 It is recommended that you back up all the data before proceeding with the upgrade to Red Hat Enterprise Linux 7. This data might be quite large.
```

Comment:
Hm, this file seems to mix and match migration vs. upgrade. @pirat89, isn't this module useful just for migration?

Reply:
@mbocek: It is informational module. The path in `kickstastart` is used just because it is only directory where these files can be stored and kept inside the tarball.  It is related to the https://github.com/upgrades-migrations/preupgrade-assistant/issues/181 RFE for P-A.